### ISSUE TITLE
TypeScript typings not found: missing "tyipings/types" declaration in `package.json`?

### ISSUE THREAD

**Issue by headcr4sh** [ceifa/steamworks.js#19]
I tried to include the `steamworks.js` module in my TypeScript-based codebase.
Unfortunately, Visual Studio Code / the Type Script Compiler complains:

> Could not find a declaration file for module 'steamworks.js'.
> '/[PATH_TO_MY_PROJECT]/node_modules/steamworks.js/index.js' implicitly has an 'any' type.
> Try `npm i --save-dev @types/steamworks.js` if it exists or add a new declaration (.d.ts) file containing `declare module 'steamworks.js';`ts(7016)

I assume that the file `client.d.ts` must be referenced in `package.json`:

```json
{
  "name": "steamworks.js",
  "types": "client.d.ts",
  "REST_OF_PACKAGE_JSON": "AS_USUAL"
}
```

See: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

**Comment by ceifa** [comment]
Weird, it should work well since we are using JSDoc on the main file.
We cannot add the `"types": "client.d.ts"`, because it doesn't reflect the main file types.

**Comment by headcr4sh** [comment]
Mh... ok.
Thanks for the quick reply.

I'll double-check my `tsconfig.json` to see if I did something weird.

**Comment by headcr4sh** [comment]
I tried to reproduce the issue with a minimal example.
I created a Gist: https://gist.github.com/headcr4sh/3dbb43d6d094c7d341e33b0d2bb36dc8

Checking out the Gist and running `npm install && npm run compile` should show the error.

Am I missing something?

(Thanks for your support in advance! If there is anything else to help clarify my issue, let me know.)

**Comment by headcr4sh** [comment]
I just finished a PR which contains a possible solution for my problem...
... I hope this clarifies my problem and the solution that I am aiming at. ;-)

**Comment by ceifa** [comment]
Fixed in #20
